core/pxt_breakpoint.js

```diff
@@ -135,7 +135,7 @@ Blockly.Breakpoint.prototype.setVisible = function(visible) {
  * @return {boolean} True if the breakpoint is Set.
  */
 Blockly.Breakpoint.prototype.isVisible = function() {
-  return !!this.set_ && Blockly.Breakpoint.superClass_.isVisible.call(this);
+  return !!this.set_
```

Comment:
```suggestion
  return !!this.set_;
```

Reply:
nitpick

Reply:
I knew you'd catch that. I was just too lazy to add a commit!